de.metas.adempiere.adempiere/base/src/main/java/de/metas/user/UserId.java

```diff
@@ -124,4 +124,9 @@ public boolean isRegularUser()
 	{
 		return !isSystemUser();
 	}
+
+	public static UserId ofNullableRepoId(final int repoId)
```

Comment:
why not using `ofRepoIdOrNull` like we have it everywhere (e.g. BPartnerId)?

Reply:
bc in this case, ofRepoIdOrNull is not working like everywhere else and returns 0 instead of null if parameter ```repoId``` is 0.

Reply:
AFAU the gist is
> the problem is that we then have two method that mostly do the same. and it it's not clear from the name which does what (i didn't yet check the whole code)